src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/SDSEarlyReleaseDefaultingRulesService.kt

```diff
@@ -160,18 +155,7 @@ class SDSEarlyReleaseDefaultingRulesService(
     ).sentenceCalculation.adjustedDeterminateReleaseDate
 
     if (dates.containsKey(ReleaseDateType.TUSED) && !latestReleaseDate.isAfterOrEqualTo(trancheConfiguration.trancheOneCommencementDate)) {
```

Comment:
just lifted out nested functionality into conditions, thought it made it more readable